test/corpus/select.txt

```diff
@@ -26,95 +26,280 @@ LEFT OUTER JOIN t7 ON t.a = t7.a;
         (dotted_name
           (identifier)
           (identifier))
+        (interval_expression
+          (string
+            (content)))
         (alias
-          (interval_expression
-            (string
-              (content)))
           (identifier))))
     (from_clause
-      (alias
+      (join_clause
```

Comment:
Can you argue why it's supposed to be a nested structure and not a flat one?

Reply:
Hi @m-novikov , do u have any further question?